module1/task1.py

```diff
@@ -0,0 +1,9 @@
+for x in range(1, 100):
+    if x % 3 == 0 and x % 5 == 0:
+        print("FizzBazz")
+    elif x % 3 == 0:
+        print("Fizz")
+    elif x % 5 == 0:
+        print("Bazz")
+    else:
+        print(x)
```

Comment:
Looks good in scope of this task. Sometimes you'd need to make calculations/logic before dealing with results (print in that case), so one more example how your program might look like:

for x in range(1, 100):
    match_str = ""
    if x % 3 == 0:
        match_str += "Fizz"
    if x % 5 == 0:
        match_str += "Bazz"
    print(match_str or x)

Reply:
ok, got it